04_verify.sh

```diff
@@ -227,12 +227,5 @@ if [[ "${BMO_RUN_LOCAL}" == true ]] || [[ "${CAPBM_RUN_LOCAL}" == true ]]; then
   echo ""
 fi
 
-#Verify Ironic containers are running
-for name in ironic ironic-inspector dnsmasq httpd mariadb; do
```

Comment:
httpd is still running on the host, we could keep the check for this container

Reply:
Good point, added the check back